packages/marked-mathjax/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@prairielearn/marked-mathjax",
+  "version": "1.0.0",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PrairieLearn/PrairieLearn.git",
+    "directory": "packages/marked-mathjax"
+  },
+  "private": true,
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "tsc"
+  },
+  "devDependencies": {
+    "@prairielearn/tsconfig": "workspace:^",
+    "marked": "^15.0.7",
+    "mathjax-full": "^3.2.2",
```

Comment:
This package is 34MB. Say more about why we need this and not just the normal `mathjax` dependency we use elsewhere?

Reply:
At this point it's mostly for typing, which is not available in `mathjax`. The `@types/mathjax` package is for the previous version of mathjax and is not suitable here.

A bigger extent of `mathjax-full` will become necessary if/when we implement server-side. If you'd prefer to push this for when it's actually needed I can remove this dependency and have MathJax be untyped (`any`) or copy just a subset of relevant types around. I can also take a brief look at other options, but I tried before and didn't find much.

Reply:
I'm happy to either have it be untyped, or just inline the relevant part.

It's still not clear to me why `mathjax-full` is needed on the server. The normal `mathjax` package specifically shows usage in Node: https://www.npmjs.com/package/mathjax#using-mathjax-components-in-a-node-application

Reply:
Indeed, I realized that later. The only minor issue (other than missing types) is that it doesn't support modules/import, so we need a `require('mathjax')` to make it work.

Reply:
I have used `any` type in 9069240.